index.ts

```diff
@@ -156,13 +164,12 @@ try {
         } by updating your \`package.json\` ` +
         `with the newly published version${multiple ? 's' : ''}:\n` +
         newTags
-          .map(
-            (tag) =>
-              '```sh\n' +
-              `${isYarn ? 'yarn add' : 'npm install'} ${tag}\n` +
-              '```',
+          .map((tag) =>
+            installCommands
+              .map((message) => '```sh\n' + `${message} ${tag}\n` + '```')
```

Comment:
```suggestion
              .map((installCommand) => '```sh\n' + `${installCommand} ${tag}\n` + '```')
```